node/core/pvf/common/src/worker/mod.rs

```diff
@@ -317,4 +317,85 @@ pub mod thread {
 			Some(*result.0)
 		}
 	}
+
+	#[cfg(test)]
+	mod tests {
+		use super::*;
+
+		#[test]
+		fn get_condvar_should_be_pending() {
+			let condvar = get_condvar();
+			let outcome = *condvar.0.lock().unwrap();
+			assert!(outcome.is_pending());
+		}
+
+		#[test]
+		fn wait_for_threads_with_timeout_return_none_on_time_out() {
+			let condvar = Arc::new((Mutex::new(WaitOutcome::Pending), Condvar::new()));
+			let outcome = wait_for_threads_with_timeout(&condvar, Duration::from_secs(1));
+			assert!(matches!(outcome, None));
```

Comment:
Here and for similar lines: please use `assert_matches` or even `assert!(outcome.is_none())` for readability.

Reply:
@slumber Thanks a lot for your time and suggestions! I've also created two new methods for wait outcome: 'is_finished' and 'is_timeout.' Please let me know if you think this is a good idea, otherwise, I will remove them.

Reply:
I agree with you, but I have decided to add this method for future use. I thought that since we already have the is_pending method, we might also need the other ones in the future. But I'm ok with removing it.

Reply:
`assert_matches` would've been sufficient.

Reply:
Methods removed and added assert_matches instead.